src/kernels/jupyter/interpreter/jupyterInterpreterStateStore.node.ts

```diff
@@ -39,7 +39,7 @@ export class JupyterInterpreterStateStore {
         }
         const memento = this.memento.get<string | undefined>(key, undefined);
         if (memento) {
-            return Uri.file(memento);
+            return Uri.parse(memento);
```

Comment:
This was causing problems with setting the jupyter interpreter. Caused by the string to URI change.

Reply:
Really sucks because if we fail to get this, we clear it. So a lot of people might have had their interpreter cleared we set a long time ago.

Reply:
S**t happens...